web_services/profile/app.js

```diff
@@ -67,21 +67,16 @@ app.post("/upload/:channelId", function (req, res) {
     channelCache.set(channelId, channel, ttl)
     console.log(`Created new channel ${channelId} with TTL ${channel.ttl}`)
   } else {
-    // existing channel. check if this channel was about to expire, but got another upload
-    if (channel.entries.size === 0) {
-      console.log(`Restoring requested TTL ${channel.ttl} for channel ${channelId}`)
-      channelCache.ttl(channelId, channel.ttl)
-    }
+    // extend TTL of the channel when got an upload
```

Comment:
This is a general change of channel service logic from having a fixed Time-to-live to a on-demand time to live.
If we do this we need to refactor other parts of the profile service code that become obsolete (e.g. handling of channel becoming empty, finalTTL related changes...)
Also the connection making code in the client is tailored around having a fixed TTL. We need to review implications on the client before releasing this.

Reply:
we will remove this change from this release to think about it more.
we will follow the approach of using longer ttl for explorer for now.
thank you.